sdk/python/packages/flet-core/src/flet_core/geolocator.py

```diff
@@ -0,0 +1,117 @@
+from enum import Enum
+from typing import Any, Optional
+from dataclasses import dataclass
+from flet_core.control import Control
+from flet_core.ref import Ref
+
+
+class LocationAccuracy(Enum):
+    LOWEST = "lowest"
+    LOW = "low"
+    MEDIUM = "medium"
+    HIGH = "high"
+    BEST = "best"
+    BESTFORNAVIGATION = "bestForNavigation"
```

Comment:
Let's separate this:
```suggestion
    BEST_FOR_NAVIGATION = "bestForNavigation"
```

Reply:
done